lib_example.py

```diff
@@ -0,0 +1,14 @@
+import my_ext
```

Comment:
Should we move this file to `vt-tv/examples` or maybe create a scripts directory?

Reply:
I'll move it to `vt-tv/examples/scripts`